dataactbroker/handlers/fileHandler.py

```diff
@@ -80,13 +82,17 @@ def getErrorReportURLsForSubmission(self, isWarning = False):
             safeDictionary = RequestDictionary(self.request)
             submissionId = safeDictionary.getValue("submission_id")
             responseDict ={}
+            sess = GlobalDB.db().session
             for jobId in self.jobManager.getJobsBySubmission(submissionId):
-                if(self.jobManager.getJobType(jobId) == "csv_record_validation"):
+                # get the job object here so we can call the refactored getReportPath
+                # todo: replace other db access functions with job object attributes
+                job = sess.query(Job).filter(Job.job_id == jobId).one()
+                if self.jobManager.getJobType(jobId) == "csv_record_validation":
```

Comment:
Great point about prefetch. Per our convo in Slack, I updated the `Job` model to do an eager load of job type, file type, and job status info, since we almost always want one of those pieces of info when querying a job.